### PR TITLE
Fluff signs

### PR DESCRIPTION
**Pull request by unid15**
MUH FIREFLY

different color scheme because it's space china

### CODE REVIEW

**Comment by SkowronX** [comment]
Moon runes go away. Make russo signs instead

**Comment by unid15** [comment]
but chyna is a space superpower and also sol common is based on mandarin

**Comment by kol1th** [comment]
I love you @unid15 

I'm going to make vaults at some point. Or away missions. 🤔

**Comment by SkowronX** [comment]
sorry I dont speak orange

**Comment by Exxion** [comment]
Out of curiosity, do those actually say anything, or is it just supposed to _look_ like they do?

**Comment by unid15** [comment]
they're all real mandarin and most of them are even grammaticaly correct (except maybe for departmental ones because they were made with the assistance of google translate™)

**Comment by Intigracy** [comment]
Middle one clearly says no humans allowed.

Das raycisst.

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -241,4 +241,42 @@
 /obj/structure/sign/crime
 	name = "CRIME DOES NOT PAY"
 	desc = "A warning sign which suggests that you reconsider your poor life choices."
-	icon_state = "crime"
\ No newline at end of file
+	icon_state = "crime"
+
+/obj/structure/sign/chinese
+	name = "Imcomprehensible Sign"
```
Comment (PJB3005) [inline code/game/objects/structures/signs.dm]: Remove capitalization.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/signs.dm (hunk 1 above)
Comment (Exxion) [reply code/game/objects/structures/signs.dm]: That's the only problem you see with this line?

**Comment by RemieRichards** [comment]
This is cute.

**Comment by Zth--** [comment]
Reminds me of that team fortress 2 map

File: code/game/objects/structures/signs.dm (hunk 1 above)
Comment (Intigracy) [reply code/game/objects/structures/signs.dm]: The meaning of that first word is a little bit incomprehensible but other than that I dunno

Comment (PJB3005) [reply code/game/objects/structures/signs.dm]: You're implying I actually proof read code instead of nitpicking on low hanging fruit. Hah!

**Comment by unid15** [comment]
Last 3 signs

**Comment by ComicIronic** [comment]
It would be cool if you extended examine so that if someone actually has Sol Common as a language, they would be able to read the sign. These are fine as is, though.

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**